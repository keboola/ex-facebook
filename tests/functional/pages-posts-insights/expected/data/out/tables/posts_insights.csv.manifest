{"primary_key":["id","parent_id","key1","key2"],"columns":["id","ex_account_id","fb_graph_node","parent_id","name","key1","key2","value","period","title","description"]}