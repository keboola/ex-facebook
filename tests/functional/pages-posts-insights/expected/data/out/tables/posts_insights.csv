"133353749860945_122093300126070033/insights/post_negative_feedback/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_negative_feedback","","","0","lifetime","Lifetime Negative Feedback","Lifetime: The number of times people have given negative feedback to your post. (Total Count)"
"133353749860945_122093300126070033/insights/post_engaged_users/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_engaged_users","","","2","lifetime","Lifetime Engaged Users","Lifetime: The number of unique people who engaged in certain ways with your Page post, for example by commenting on, liking, sharing, or clicking upon particular elements of the post. (Unique Users)"
"133353749860945_122093300126070033/insights/post_impressions_fan/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_impressions_fan","","","0","lifetime","Lifetime Post Impressions by people who have liked your Page","Lifetime: The number of impressions of your Page post to people who have liked your Page. (Total Count)"
"133353749860945_122093300126070033/insights/post_impressions_paid/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_impressions_paid","","","0","lifetime","Lifetime Post Paid Impressions","Lifetime: The number of times your Page's post entered a person's screen through paid distribution such as an ad. (Total Count)"
"133353749860945_122093300126070033/insights/post_impressions/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_impressions","","","2","lifetime","Lifetime Post Total Impressions","Lifetime: The number of times your Page's post entered a person's screen. Posts include statuses, photos, links, videos, Reels and more. (Total Count)"
"133353749860945_122093300126070033/insights/post_negative_feedback/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_negative_feedback","","","0","lifetime","Lifetime Negative Feedback","Lifetime: The number of times people have given negative feedback to your post. (Total Count)"
"133353749860945_122093300126070033/insights/post_engaged_users/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_engaged_users","","","2","lifetime","Lifetime Engaged Users","Lifetime: The number of unique people who engaged in certain ways with your Page post, for example by commenting on, liking, sharing, or clicking upon particular elements of the post. (Unique Users)"
"133353749860945_122093300126070033/insights/post_impressions_fan/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_impressions_fan","","","0","lifetime","Lifetime Post Impressions by people who have liked your Page","Lifetime: The number of impressions of your Page post to people who have liked your Page. (Total Count)"
"133353749860945_122093300126070033/insights/post_impressions_paid/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_impressions_paid","","","0","lifetime","Lifetime Post Paid Impressions","Lifetime: The number of times your Page's post entered a person's screen through paid distribution such as an ad. (Total Count)"
"133353749860945_122093300126070033/insights/post_impressions/lifetime","133353749860945","page_feed_insights","133353749860945_122093300126070033","post_impressions","","","2","lifetime","Lifetime Post Total Impressions","Lifetime: The number of times your Page's post entered a person's screen. Posts include statuses, photos, links, videos, Reels and more. (Total Count)"
